notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -4395,6 +4395,52 @@ law enforcement [A police detective was investigating a possible murder.]
 sexual debut [The college freshmen Roger was deflowered by a grotesque alien that had taken on the appearance of beautiful a college girl.]
 
 
+movie: Ghost in the Shell 2: Innocence (2004)
+=============================================
+
+:: Title
+Ghost in the Shell 2: Innocence
+
+:: Description
+Ghost in the Shell 2: Innocence, known in Japan as just Innocence (イノセンス,
+Inosensu), is a 2004 Japanese animated cyberpunk film written and directed by
+Mamoru Oshii. The film serves as a standalone sequel to Oshii's 1995 film
+Ghost in the Shell and is loosely based on the manga by Masamune Shirow.
+
+Synopsis: A cyborg detective and his partner are tasked to investigate what's
+behind a recent rise in cases of doll-like sex robots murdering their owners.
+
+:: Date
+2004-03-06
+
+:: References
+https://en.wikipedia.org/wiki/Ghost_in_the_Shell_2:_Innocence
+
+:: Ratings
+3 <paul>
+
+:: Major Themes
+cyborg [The main character Batou had a "cyberbrain", artificial arm of some kind, and robotic eyes. In general, the film is set in a future where being a cyborg is no big deal.]
+sex-droid [The story concerns malfunctioning doll-like sex robots killing their owners.]
+murder [The police detectives Batou and Togusa were investigating a number of murders by malfunctioning doll-like sex robots.]
+law enforcement [The story follows the police detectives Batou and Togusa as they investigate a number of enigmatic murders.]
+human interpersonal relationship [The story explores the relationship between the cyborg police detective Batou and his new mostly human partner Togusa.]
```

Comment:
Buddy-cops is too specific but maybe "buddy co-workers" would be useful 🤔

Reply:
We are building up a pretty long list work partners (almost exclusively in the context of law enforcement) under the meta-theme **human interpersonal relationship**.